mkdocs.yml

```diff
@@ -0,0 +1,115 @@
+site_name: emodpy
+site_url: https://docs.idmod.org/projects/emodpy/en/latest/
```

Comment:
This is only the URL when hosted on RTD. This should instead be the github.io URL. We will set up a redirect so that emod.idmod.org/emodpy will point to that github.io URL.